recipes/reago/meta.yaml

```diff
@@ -13,9 +13,19 @@ requirements:
   build:
     - python
     - setuptools
+    - os
```

Comment:
Sorry for uploading it faulty (I was switching machines), it does indeed fail locally too.
So, I only need to include Networkx as a dependency, and leave the rest out?